src/main/resources/import.sql

```diff
@@ -0,0 +1,138 @@
+-- phpMyAdmin SQL Dump
```

Comment:
Curious about this choice, and what everyone else thinks. can't we have a method that populates the db with data for us at each run (we can comment it out when we don't need it)
rather than importing an SQL script and running on the db through the java `DatasourceConfiguration` class

Reply:
Yes, there are multiple ways to create dummy data and chose this one cause we can execute the whole SQL script and it seems easy.